x-pack/filebeat/docs/inputs/input-httpjson.asciidoc

```diff
@@ -566,6 +566,11 @@ Required if using split type of `string`.  This is the sub string used to split
 
 Valid when used with `type: map`. When not empty, defines a new field where the original key value will be stored.
 
+[float]
+==== `response.split[].ignore_error`
```

Comment:
As it behaves right now, I would expect it to ignore all kind of errors, such as https://github.com/elastic/beats/pull/27880/files#diff-5d902f14f54910145f27c89c42ed581f8c81ffea88cd3cb03cf1f2b86f4ff6a8R127. I am fine with the current behavior of just ignoring empty values, but would suggest then renaming it to something on the lines of `ignore_empty`, similar to other processors. WDYT?

Reply:
I was thinking pretty hard about that, which is why I changed the internal field's name to `ignoreEmptyFieldError`, but you're right. There are errors that it won't ignore as it stands; anything that is non-nil and not `common.ErrKeyNotFound` (which is arguably empty) causes an error return at L106.

There is an `ignore_empty_value` flag used in the httpjson cursor processor and elsewhere, so this is probably a good name to use.